src/v3/src/transformer/email/emailVerificationTransformer.ts

```diff
@@ -53,17 +54,16 @@ export const transformEmailVerification: IdxStepTransformer = (transaction, form
     type: 'Description',
     options: {
       content: redactedEmailAddress
-        ? 'next.email.verify.subtitleWithEmailAddress'
-        : 'next.email.verify.subtitleWithoutEmailAddress',
-      contentParams: [
-        redactedEmailAddress,
-      ],
+        ? loc('oie.email.verify.subtitleWithEmailAddress', 'login', [redactedEmailAddress])
+            .replace('<$1>', '')
+            .replace('</$1>', '')
```

Comment:
I thought `loc` handled these

Reply:
:( It doesn't unfortunately, it gives back these characters

Reply:
how about wrap loc into a util module, so corner cases like this can be handled in a centralized place?